feedbag/feedbag.go

```diff
@@ -34,7 +35,9 @@ func Start(port, templatesDir string) error {
 
 	// Process our templates
 	TemplatesDir = templatesDir
-	Templates = getTemplates(TemplatesDir)
+	var err error
+	Templates, err = template.ParseDir(TemplatesDir)
```

Comment:
why are we pre-declaring `err` instead of just doing `Templates, err := getTemplates()`